migrations/versions/900_add_brief_is_a_copy.py

```diff
@@ -0,0 +1,22 @@
+"""Add Brief.is_a_copy boolean, default False, nullable False
+
+Revision ID: 890
+Revises: 880
+Create Date: 2017-06-01 11:24:53.346954
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '900'
+down_revision = '890'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('briefs', sa.Column('is_a_copy', sa.Boolean(), server_default=sa.text(u'false'), nullable=False))
```

Comment:
Why have we got `sa.text(u'false')` here? In e.g. migration #840 we just have `server_default='false'`.

It runs fine as it is (at least looks like it as far as I can tell), but just wondering if there's a reason for the `sa.text()` wrapper?

To be honest both of these seem weird because they're using string/text-types rather than a Boolean, but it clearly works ¯\\_(ツ)_/¯